airbyte-integrations/connectors/source-github/source_github/schemas/workflow_jobs.json

```diff
@@ -24,21 +30,25 @@
       "type": "string"
     },
     "html_url": {
-      "type": ["string", "null"]
+      "type": ["null", "string"]
     },
     "status": {
       "type": "string"
     },
     "conclusion": {
-      "type": ["string", "null"]
+      "type": ["null", "string"]
+    },
+    "created_at": {
+      "type": ["null", "string"],
+      "format": "date-time"
     },
     "started_at": {
-      "format": "date-time",
-      "type": "string"
+      "type": "string",
+      "format": "date-time"
     },
     "completed_at": {
-      "format": "date-time",
-      "type": ["string", "null"]
+      "type": ["null", "string"],
+      "format": "date-time"
```

Comment:
Any other (old or new) date strings that we should add a `format` for? Can you remind me how this is used when in a schema vs in a spec? (In a spec its used for the frontend to create a date-time-picker for the user to use)

Reply:
in stream schema `"format": "date-time"` is used for normalization to create on SQL databases DATETIME column (not STRING column)

BTW, here I just moved `format` below "type" - it's  cosmetic change, I did not add any new formats

Reply:
> in stream schema "format": "date-time" is used for normalization to create on SQL databases DATETIME column (not STRING column)

Aha, that makes sense. I'm not super familiar with normalization. Thanks!